lte/gateway/python/integ_tests/s1aptests/test_attach_detach_maxbearers_twopdns.py

```diff
@@ -128,20 +128,21 @@ def test_attach_detach_maxbearers_twopdns(self):
 
             print("Sleeping for 5 seconds")
             time.sleep(5)
-            for i in range(loop):
+            num_flows_per_bearer = 4
+            for idx in range(loop):
```

Comment:
Can we rename `loop` to a more intuitive variable name, such as `num_dedicated_bearers`?